composer.json

```
@@ -1,100 +1,75 @@
-  },
-  "require-dev": {
-    "friendsofphp/php-cs-fixer": "^3.1",
-    "justinrainbow/json-schema": "^5.2",
-    "php-parallel-lint/php-parallel-lint": "^1.3",
-    "phpstan/phpstan": "^0.12.99",
-    "phpstan/phpstan-beberlei-assert": "^0.12.6",
-    "phpstan/phpstan-phpunit": "^0.12.22",
-    "phpunit/phpunit": "^9.5",
-    "rector/rector": "^0.11.56",
-    "thecodingmachine/phpstan-safe-rule": "^1.0"
-  },
-  "config": {
-    "preferred-install": "dist",
-    "sort-packages": true
-  },
-  "minimum-stability": "dev",
-  "prefer-stable": true,
-  "scripts": {
-    "security:check": [
-      "bin/local-php-security-checker"
-    ],
-    "code-style:fix": [
-      "PHP_CS_FIXER_FUTURE_MODE=1 php-cs-fixer fix --config='./tools/php-cs-fixer/config.php' --diff --ansi --using-cache=no"
-    ],
-    "code-style:check": [
-      "PHP_CS_FIXER_FUTURE_MODE=1 php-cs-fixer fix --config='./tools/php-cs-fixer/config.php' --dry-run --diff --ansi --using-cache=no"
-    ],
-    "lint:php": [
-      "parallel-lint src",
-      "parallel-lint tests"
-    ],
-    "lint": [
-      "@lint:php"
+    "type": "library",
+    "name": "treblle/treblle-php",
+    "description": "Stay in tune with your APIs",
+    "homepage": "https://treblle.com/",
+    "keywords": [
+        "phpunit",
+        "treblle",
+        "api",
+        "monitoring",
+        "debuging",
+        "documentation"
     ],
-    "rector:check": [
-      "rector --config='./tools/rector/config.php' --dry-run --ansi --no-progress-bar"
+    "license": "MIT",
+    "authors": [
+        {
+            "role": "Developer",
+            "name": "Vedran Cindrić",
+            "email": "vedran@treblle.com",
+            "homepage": "https://treblle.com/"
+        },
+        {
+            "role": "Developer",
+            "name": "Steve McDougall",
+            "email": "juststevemcd@gmail.com",
+            "homepage": "https://treblle.com/"
+        }
     ],
-    "rector:fix": [
-      "rector  --config='./tools/rector/config.php' --ansi --no-progress-bar"
-    ],
-    "phpstan": [
-      "phpstan analyse --configuration='./tools/phpstan/config.neon' --ansi --no-progress --memory-limit=3G"
-    ],
-    "static-analysis": [
-      "@composer validate",
-      "@lint",
-      "@code-style:check",
-      "@phpstan",
-      "@rector:check"
-    ],
-    "phpunit": [
-      "phpunit --configuration='./tools/phpunit/config.xml' --colors --do-not-cache-result"
-    ],
-    "tests": [
-      "@phpunit"
-    ],
-    "qc": [
-      "@static-analysis",
-      "@tests"
-    ]
-  }
+    "autoload": {
+        "psr-4": {
+            "Treblle\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Tests\\Treblle\\": "tests/"
+        }
+    },
+    "require": {
+        "php": "^8.1",
+        "php-http/client-common": "^2.6",
+        "php-http/discovery": "^1.15.2",
+        "php-http/mock-client": "^1.5",
+        "treblle/treblle-sdk-core": "0.0.2"
```

To make updating it easier, the constraint could be less restrictive

```suggestion
        "treblle/treblle-sdk-core": "^0.0"
```

---

CI is now failing so maybe `"treblle/treblle-sdk-core": "^0.0.2"` is a better idea